website/docs/docs/collaborate/explore-projects.md

```diff
@@ -0,0 +1,142 @@
+- To view test results, you must run `dbt test` or `dbt build` within a job in the environment.
+- To view source freshness check results, you must run `source freshness` within a job in the environment.
+
+The need to run these commands will diminish and richer, more timely metadata will become available as the Discovery API and its underlying platform evolve.
+
+:::tip Join the beta
+
+dbt Explorer is a [beta feature](/docs/dbt-versions/product-lifecycles#dbt-cloud) and subject to change without notification. More updates to this feature coming soon.
+
+If you’re interested in joining the beta, please contact your account team.
+ 
+:::
+
+## Prerequisites
+
+- You have a [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) or AWS single-tenant dbt Cloud account on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
+- You have set up a [production deployment environment](/docs/deploy/deploy-environments#set-as-production-environment-beta) for each project you want to explore.
+    - There has been at least one successful job run in the production deployment environment.
+- You are on the dbt Explorer page.
+    - To go to the page, select **Explore (Beta)** from the top navigation bar in dbt Cloud.
+
+## Explore the project’s lineage
+
+dbt Explorer provides a visualization of your project’s <Term id="dag">DAG</Term> that you can interact with. To start, select **Overview** in the left sidebar and click the **Explore Lineage** button on the main (center) section of the page. 
+
+If you don't see the lineage graph immediately, click **Render Lineage**. It can take some time for the graph to render depending on the size of your project and your computer’s available memory. The graph of very large projects might not render so, instead, you can select a subset of nodes by using selectors.
+
+The nodes in the lineage graph represent the project’s resources and the edges represent the relationships between the nodes. Resources like tests and macros display in the lineage within their [resource details pages](#view-resource-details) but not within the overall project lineage graph. Nodes are color-coded and include iconography according to their resource type.
+
+To interact with the lineage graph, you can:
+
+- Hover over any item in the graph to display the resource’s name and type.
+- Zoom in and out on the graph by mouse-scrolling.
+- Grab and move the graph.
+- Click on a resource to highlight its relationship with other resources in your project.
+- [Search and select specific resources](#search-resources) or a subset of the DAG using selectors and lineage (for example, `+[YOUR_RESOURCE_NAME]` displays all nodes upstream of a particular resource).
+- [View resource details](#view-resource-details) by selecting a node in the graph (double-clicking).
+
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/lineage-v1.gif" title="Explore the lineage graph" />
```

Comment:
i'd try making this bigger, it looks pretty small in the preview
![Screenshot 2023-09-13 at 16 55 35](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/9d2a40c9-b402-4e7a-8cab-c1eee8e895af)

```suggestion
<Lightbox src="/img/docs/collaborate/dbt-explorer/lineage-v1.gif" width="85%" title="Explore the lineage graph" />
```

Reply:
yeah, i agree and was playing around with the `width` property yesterday. but i decided to leave it as is because for some reason the title is no longer centered with the GIF when i add that property:

<img width="853" alt="Screenshot 2023-09-13 at 9 32 03 AM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/f2ec1555-e74d-44cb-af14-dcad2f7a4008">

i wonder if this behavior is because my original source was an MP4 and i used a converter turn it into a GIF. for phase 2 (in ~several weeks) we'll be recording in native GIF so thinking this issue will resolve itself